tests/unit/test_context7_service_extended.py

```
@@ -35,14 +37,16 @@ def test_release_lock_error(self, temp_project_dir, capsys):
 
         captured = capsys.readouterr()
         # Should log warning but not raise
-        assert "Error releasing lock" in captured.out or True  # Logger might not output to stdout
+        assert (
+            "Error releasing lock" in captured.out or True
```

Pylint (3.3.7)

[refactor] Boolean condition ''Error releasing lock' in captured.out or True' will always evaluate to 'True'

(R1727)